daemon/src/GHCSpecter/Server/Types.hs

```diff
@@ -161,7 +165,12 @@ data ServerState = ServerState
   , _serverShouldUpdate :: Bool
   , _serverInbox :: Inbox
   , _serverSessionInfo :: SessionInfo
-  , _serverTiming :: Map ModuleName Timer
+  , _serverDriverModuleMap :: IntMap ModuleName
```

Comment:
we had better make a single type for this bimap (map and reverse map) and hide bare Int type from the interface for safety.

Reply:
KeyMap and BiKeyMap are introduced.